packages/loot-core/src/shared/util.ts

```diff
@@ -310,6 +311,10 @@ export function amountToCurrency(n) {
   return getNumberFormat().formatter.format(n);
 }
 
+export function amountToCurrencyNoDecimal(n) {
+  return getNumberFormat(0).formatter.format(n);
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏no need to define a new parameter if we can use something that already exists
```suggestion
  return getNumberFormat({ hideFraction: true }).formatter.format(n);
```

Reply:
Actually.. why are we even overriding the `hideFraction` value? The user has a choice in UI whether to see fractions or not. IMO we should stick to whatever the customer has chosen here. Which makes the entire `amountToCurrencyNoDecimal` util redundant as then we can use the existing `amountToCurrency`

<img width="580" alt="Screenshot 2024-01-11 at 19 16 45" src="https://github.com/actualbudget/actual/assets/886567/7e111214-4e08-4039-9db4-285e2d85310d">

Reply:
With the decimals included on the bar graph they hang way over into the neighboring bars so to make them fit the font had to be unreadably small (especially for numbers above 1k). Plus the "amountToCurrency" is displayed in the hover tooltip. It seemed like this was the best way to display this data on these graphs so that they are easy to read and give the best experience. I'm happy to use the number with decimals but wanted to offer a different opinion and explain my logic for why I didn't originally use them.

Reply:
Good point. Lets follow your recommendation :)